frontend/src/features/login/components/LoginForm.tsx

```diff
@@ -107,18 +107,43 @@ export function LoginForm() {
                 {isSubmitting ? 'Logging in...' : 'Log In'}
               </Button>
 
-              <Typography variant="body2" align="center" sx={{ mt: 2 }}>
-                Don&apos;t have an account?{' '}
-                <Link component={NextLink} href="/signup">
-                  Sign up
-                </Link>
-              </Typography>
-              <Typography variant="body2" align="center" sx={{ mt: 2 }}>
+              {!isVendorLogin && (
```

Comment:
Seeing this makes me think it'll be easier to maintain if we have reusable components for the shared login elements above, and then separate parent elements for the login form (i.e. LoginForm and VendorLoginForm)

Reply:
Ah yeah that makes sense! I'll clean it up during tech debt week since that is done in a few places!!